test_queue.py

```diff
@@ -12,20 +13,29 @@ def populated_queue(request):
     return q
 
 
```

Comment:
Every test deserves a docstring - Write every test like you'd be debugging this code 2 years from now with no recollection of any specifics of your code.